### PR TITLE
Create docker configuration production

### PR DESCRIPTION
**Pull request by criticic**
Fixes #

#### Describe the changes you have made in this pr -
I mostly used existing docker configuration and changed them according to production commands given in readme, 

To run : ```docker-compose -f docker-compose.prod.yml up```

### CODE REVIEW

File: Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.6.5
+
+# set up workdir
+RUN mkdir /circuitverse
+WORKDIR /circuitverse
+
+# install dependencies
+RUN apt-get update -qq && apt-get install -y imagemagick && apt-get clean
+
+RUN wget -qO- https://deb.nodesource.com/setup_8.x | bash - && apt-get install -y nodejs && apt-get clean
```
Comment (tachyons) [inline Dockerfile.prod]: node 8 is no longer supported

Comment (fluctix) [reply Dockerfile.prod]: I still used node 8 😔. gotta upgrade

**Comment by fluctix** [comment]
@tachyons @satu0king @alanjsph please review...

File: docker-compose.prod.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    build: .
+    command: bundle exec sidekiq -e production -q default -q mailers -d -L tmp/sidekiq.log
+    volumes:
+      - .:/circuitverse
+      - ./config/database.docker.yml:/circuitverse/config/database.yml
+    environment:
+      REDIS_URL: "redis://redis:6379/0"
+  web:
+    build: .
+    entrypoint: /circuitverse/bin/docker_run
```
Comment (tachyons) [inline docker-compose.prod.yml]: this docker run command is for development environment only, as it run db:seed file

Comment (satu0king) [reply docker-compose.prod.yml]: @thiefie, you need to make changes to `/circuitverse/bin/docker_run` file.

**Comment by thiefie** [comment]
I was a bit busy, i will try to solve the issue soon.

**Comment by satu0king** [comment]
@criticic will you be working on this?

**Comment by wallento** [comment]
Wouldn't `db:seed` have no effect on initialized production systems?

**Comment by tachyons** [comment]
Done in slightly different manner

**Pull request closed without merging**